hw2/Matrix.java

```diff
@@ -26,6 +24,7 @@ public static void main(String args[])
     //C = A * B
     for (i=0; i<n; i++) {
       for (j=0; j<n; j++) {
+        c[i][j] = 0;
```

Comment:
It's probably better to be able to see the matrix C has been initialized with zero values when the values for the matrix C is calculated. This reduces a dependency on other part of code as if the initialization code at line 20 changed then the result of the matrix C may change.